### PR TITLE
chore: prepare v2.0.0-rc.1 release

### PR DESCRIPTION
**Pull request by slorber**
chore: prepare v2.0.0-rc.1 release

### CODE REVIEW

File: admin/publish.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Get access from the Docusaurus npm admins (@yangshun/@JoelMarcey).
 
 You need publish access to **the main Docusaurus repository** (not a fork).
 
-## npm
+## NPM
```
Comment (Josh-Cena) [inline admin/publish.md:13]:
```suggestion
## npm
```

npm is lowercased, no matter where.

Comment (Josh-Cena) [inline admin/publish.md:13]: Revert changes in this file?

Comment (slorber) [reply admin/publish.md:13]: oh shit this is totally unwanted changes, thanks 👍

**Comment by Jess3242** [comment]
> ### <span aria-hidden="true">✅</span> [V2]
> 
> 
> |  Name | Link |
> |---------------------------------|------------------------|
> |<span aria-hidden="true">🔨</span> Latest commit | c8ddd02a8e68dfaf515c20465a049a83153bd205 |
> |<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/docusaurus-2/deploys/62d03f1328c57a0009ac6da1 |
> |<span aria-hidden="true">😎</span> Deploy Preview | https://deploy-preview-7778--docusaurus-2.netlify.app |
> |<span aria-hidden="true">📱</span> Preview on mobile | <details><summary> Toggle QR Code... </summary><br /><br />QR Code<br /><br />_Use your smartphone camera to open QR code link._</details> |
> ---
> 
> _To edit notification comments on pull requests, go to your [Netlify site settings](https://app.netlify.com/sites/docusaurus-2/settings/deploys#deploy-notifications)._

**Pull request merged**